.devcontainer/postCreate.sh

```
@@ -0,0 +1,233 @@
+COUCHDB_USRPWD=${COUCHDB_USER}:${COUCHDB_PASSWORD}
+COUCHDB_PORT=5984
+
+export SRV=http://localhost:${COUCHDB_PORT}
+export KEYCLOAK=http://localhost:8090
+
+
+echo Couch $SRV
+echo keycloak $KEYCLOAK
+
+#SYSTEM databases
+curl -u ${COUCHDB_USRPWD} -X PUT ${SRV}/_users
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)